### DISCUSSION TITLE
rtl truncate support

### DISCUSSION THREAD

**Discussion by nh-tu-z** [tailwindlabs/tailwindcss#14051] [Help] [answered]
I find a [discussion](https://github.com/tailwindlabs/tailwindcss/discussions/7876 ) before about truncate rtl support. And I can't find a proper answer for this question. 
So regarding to this [example](https://codepen.io/carusog/pen/MBeeZM), I want to truncate the text like:

`<p className='rtl:truncate'>This is truncated message at the left side.</p>`

My expectation is **"... at the left side."** ([reference docs](https://tailwindcss.com/docs/hover-focus-and-other-states#rtl-support)). 

But it does not work as my expectation. 

Does tailwind latest version support this?

**Comment by wongjn** [comment, marked as answer]
Seems to work in [this Tailwind Play](https://play.tailwindcss.com/gSSFUVewTy) which would suggest it could be something particular to your project which makes it not work as you intend.
